ddl/placement_policy_test.go

```diff
@@ -190,10 +180,10 @@ func (s *testDBSuite6) TestPlacementPolicy(c *C) {
 		"PRIMARY_REGION=\"cn-east-1\" " +
 		"REGIONS=\"cn-east-1,cn-east-2\" ")
 	tk.MustQuery("show warnings").Check(testkit.Rows("Note 8238 Placement policy 'X' already exists"))
-
+	dom.DDL().SetHook(originalHook)
```

Comment:
I think it's better to remove this line and init custom hook like this:

```
hook := &ddl.TestDDLCallback{Do: dom}
```

In this way, the custom hook can help reload schema automatically

Reply:
Fixed.